episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+2024-05-24 16:28:16 - context        - INFO     - ---------------------------------------------
+2024-05-24 16:28:16 - context        - INFO     -  Welcome to
+2024-05-24 16:28:16 - context        - INFO     -                   _                     __
+2024-05-24 16:28:16 - context        - INFO     -                  | |                   / /
+2024-05-24 16:28:16 - context        - INFO     - __   ____ _ _ __ | |_ __ _  __ _  ___ / /_
+2024-05-24 16:28:16 - context        - INFO     - \ \ / / _` | '_ \| __/ _` |/ _` |/ _ \ '_ \
+2024-05-24 16:28:16 - context        - INFO     -  \ V / (_| | | | | || (_| | (_| |  __/ (_) |
+2024-05-24 16:28:16 - context        - INFO     -   \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/
+2024-05-24 16:28:16 - context        - INFO     -                             __/ |
+2024-05-24 16:28:16 - context        - INFO     -                            |___/
+2024-05-24 16:28:16 - context        - INFO     -
+2024-05-24 16:28:16 - context        - INFO     -  --> Join us on Discord! https://discord.gg/rwRvwyK
+2024-05-24 16:28:16 - context        - INFO     -  --> Docs: https://docs.vantage6.ai
+2024-05-24 16:28:16 - context        - INFO     -  --> Blog: https://vantage6.ai
+2024-05-24 16:28:16 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 16:28:16 - context        - INFO     - Cite us!
+2024-05-24 16:28:16 - context        - INFO     - If you publish your findings obtained using vantage6,
+2024-05-24 16:28:16 - context        - INFO     - please cite the proper sources as mentioned in:
+2024-05-24 16:28:16 - context        - INFO     - https://vantage6.ai/vantage6/references
+2024-05-24 16:28:16 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 16:28:16 - context        - INFO     - Started application vantage6
+2024-05-24 16:28:16 - context        - INFO     - Current working directory is '***/vantage6/node'
+2024-05-24 16:28:16 - context        - INFO     - Successfully loaded configuration from '***/vantage6/node/node1.yaml'
+2024-05-24 16:28:16 - context        - INFO     - Logging to '***/vantage6/node/node1/node_user.log'
+2024-05-24 16:28:16 - context        - INFO     - Common package version '4.5.0'
+2024-05-24 16:28:16 - context        - INFO     - vantage6 version '4.5.0'
+2024-05-24 16:28:16 - context        - INFO     - vantage6 version '4.5.0'
+[info ] - Your new API key has been uploaded to the config file ***/vantage6/node/node1.yaml.
+```
+
+When you finish the process, the node configuration file will be updated with the new API key.
+
+To make the new API key effective, you need to restart the node by running the command `v6 node stop` and then `v6 node start`.
+
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 4: Reset the API key of a node
+
+1. How do you reset the API key of a node? Try to reset the API key of the node you just created.
```

Comment:
do you mean resetting here or updating? :-P

Reply:
Updating is definitely the best.